locales/en-US/app.ftl

```diff
@@ -468,6 +468,12 @@ MarkerContextMenu--select-the-sender-thread =
 MarkerFiltersContextMenu--drop-samples-outside-of-markers-matching =
     Drop samples outside of markers matching “<strong>{ $filter }</strong>”
 
+MarkerCopyTableContextMenu--copy-table-as-plain =
```

Comment:
nit: Can you add a new section comment for `MarkerCopyTableContextMenu` similar to `MarkerFiltersContextMenu` above?